<launch>
  <arg name="output" default="screen"/>

  <remap from="odom" to="odometry/filtered"/>
  <!-- <remap from="cmd_vel" to="planner/cmd_vel"/> -->

  <node name="move_base"
        pkg="move_base"
        type="move_base"
        clear_params="true"
        output="$(arg output)">
    <rosparam command="load"
              file="$(find leo_navigation)/config/move_base.yaml"/>

    <!-- costmaps -->
    <rosparam command="load"
              ns="global_costmap"
              file="$(find leo_navigation)/config/costmap/costmap_common.yaml"/>
    <rosparam command="load"
              ns="local_costmap"
              file="$(find leo_navigation)/config/costmap/costmap_common.yaml"/>
    <rosparam command="load"
              file="$(find leo_navigation)/config/costmap/local_costmap.yaml"/>
    <rosparam command="load"
              file="$(find leo_navigation)/config/costmap/global_costmap.yaml"/>

    <!-- planners -->
    <rosparam command="load"
              file="$(find leo_navigation)/config/planner/base_local_planner.yaml"/>
    <rosparam command="load"
              file="$(find leo_navigation)/config/planner/base_global_planner.yaml"/>
  </node>
</launch>
